src/Language/PureScript/Parser/Declarations.hs

```diff
@@ -564,6 +571,15 @@ parseBinderNoParens = P.choice
                       , ParensInBinder <$> parens parseBinder
                       ] P.<?> "binder"
 
+parseBinderLocalDecl :: TokenParser Binder
```

Comment:
```purescript
let
  _ = foo
  123 = bar
in
  ...
```

The code above doesn't look proper to me, because it doesn't add any value declaration. Would we just allow them too?